app/experimenter/experiments/migrations/0208_alter_nimbusexperiment_channel.py

```diff
@@ -0,0 +1,29 @@
+# Generated by Django 3.2.13 on 2022-05-05 19:34
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("experiments", "0207_nimbusexperiment_languages"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="nimbusexperiment",
+            name="channel",
+            field=models.CharField(
+                choices=[
+                    ("", "No Channel"),
+                    ("default", "Unbranded"),
+                    ("nightly", "Nightly"),
+                    ("beta", "Beta"),
+                    ("release", "Release"),
+                    ("esr", "Esr"),
```

Comment:
I'm guessing "Esr" is some automatic title casing? We could name it "Extended Support Release"

Reply:
Oh yeah tehre is a way to specify a custom label but we're not doing it yet.  If we wanna do that we'd have to do it as a followup.